tools/dgeni/common/dgeni-accessors-parse.ts

```diff
@@ -0,0 +1,54 @@
+import {getTypeText} from 'dgeni-packages/typescript/services/TsParser/getTypeText';
+import {getDecorators} from 'dgeni-packages/typescript/services/TsParser/getDecorators';
+import {
+  ClassDeclaration,
+  SyntaxKind,
+  GetAccessorDeclaration,
+  SetAccessorDeclaration,
+  Symbol,
+  displayPartsToString
+} from 'typescript';
+import {CategorizedPropertyMemberDoc} from '../processors/categorizer';
+
+/**
+ * Accessors are not parsed properly within the latest dgeni-packages package.
```

Comment:
Is there an issue to link for this? What about the accessors isn't parsed correctly?

Reply:
I only reported this on Slack if I recall. Created an issue for it now: https://github.com/angular/dgeni-packages/issues/246